### PR TITLE
Fixed volume and inputs

### PR DESCRIPTION
**Pull request by ghostintranslation**
- Fixed volume out of range
- Fixed analog inputs using variables instead of plain numbers
- Little adjustment on the sustain of the envelope that I previously forgot to commit

### CODE REVIEW

File: SimpleSynth.ino [hunk 1]
```diff
@@ -53,7 +53,7 @@ void setup() {
   AudioMemory(40);
 
   sgtl5000_1.enable();
-  sgtl5000_1.volume(2);
+  sgtl5000_1.volume(0.5);
```
Comment (neophyte006) [inline SimpleSynth.ino:56]: Fullscale = 1.0 (distortion expectable) bigger values are not possible. 
0.8 corresponds to a non distorting good usage of bit-depth with enough headroom. 
0.5 is loud enough for testing purposes whilst delivering good enough sound quality.

File: Synth.h [hunk 2]
```diff
@@ -286,7 +286,6 @@ inline void Synth::update(){
   // Attack
   // TODO This should send a standardized value from 0 to 1023
   int attack = map(analogRead(this->attackPin), 0, 1023, 0, 2000);
-  attack=0;//TODO: REMOVE THAT
```
Comment (neophyte006) [inline Synth.h:289]: 2nd attack overwrites analogRead what makes it useless

File: SimpleSynth.ino [hunk 3]
```diff
@@ -20,7 +20,7 @@ bool controllerIsLaunchpad = true;
 const int interval_time = 50;
 elapsedMillis clock_count;
 
-Synth synth(0, 1, 2, 3, 4, 5, 6, 7, 8);
+Synth synth(A0, A1, A2, A3, A4, A5, A6, A7, A8);
```
Comment (neophyte006) [inline SimpleSynth.ino:23]: Replacing plain numbers by exact analog pin declaration fixes analogRead issues with Teensy 3.6

File: Voice.h [hunk 4]
```diff
@@ -134,6 +134,7 @@ inline Voice::Voice(){
   this->mixer = new AudioMixer4();
   this->mixer->gain(0, 1);
   this->envelope = new AudioEffectEnvelope();
+  this->envelope->sustain(0.5);
```
Comment (neophyte006) [inline Voice.h:137]: Todo: Add another potentiometer to adjust Sustain level

**Review by neophyte006 — APPROVED** [review]

File: Synth.h (hunk 2 above)
Comment (ghostintranslation) [reply Synth.h:289]: Yes, that was a hardcoded thing I just did to test previously and so I removed itnow

File: Voice.h (hunk 4 above)
Comment (ghostintranslation) [reply Voice.h:137]: That wouldn't be hard to add, but currently I can have only 9 potentiometer in the format I did the synth and had to choose what's controllable and what's not. Might be a later thing.

**Pull request merged**